tasks.py

```diff
@@ -39,7 +39,7 @@ def is_truthy(arg):
 namespace.configure(
     {
         "nautobot_chatops": {
-            "nautobot_ver": "1.1.6",
+            "nautobot_ver": "1.2.0",
```

Comment:
This is a mismatch from the `pyproject.toml`: https://github.com/nautobot/nautobot-plugin-chatops/blob/1dcb3cf024efc70fd2f604b3be5d11fdc9938cb3/pyproject.toml#L25

Reply:
Agreed